conda_smithy/cli.py

```diff
@@ -156,6 +157,8 @@ def __call__(self, args):
         try:
             ci_register.add_project_to_travis(owner, repo)
             ci_register.travis_token_update_conda_forge_config(args.feedstock_directory, owner, repo)
+            time.sleep(1)
```

Comment:
Yeah, there must be a better solution like reusing the Travis CI API token between calls. We should discuss it in a new issue/PR.